scv-prototype-frontend/src/app/views/Private/Profile/Names/Names.jsx

```diff
@@ -29,11 +35,13 @@ const Names = ({ names }) => {
 };
 
 Names.propTypes = {
-  names: PropTypes.arrayOf(PropTypes.shape(Name.propTypes))
+  person: PropTypes.objectOf(PropTypes.object),
+  programs: PropTypes.arrayOf(PropTypes.object)
```

Comment:
It would be better to use `PropTypes.shape` here.

Reply:
Fixed in next commit.